packages/core/cache/nuxt/index.js

```diff
@@ -28,15 +28,18 @@ function createInvalidationEndpoint (driver, options) {
         tags: Array.from(new Set(tags))
       });
 
-      response.writeHead(200);
+      response.status(200).send('Cache invalidated successfully!');
     } catch (error) {
       Logger.error('Cache driver thrown an error when invalidating cache! Operation skipped.');
       Logger.error(error);
 
-      response.writeHead(500);
+      response
+        .status(500)
+        .send(
+          'Cache driver thrown an error when invalidating cache! Operation skipped. ' +
+          error
```

Comment:
```suggestion
          error.toString()
```

WDYT?

Reply:
Corrected in [41c9831](https://github.com/vuestorefront/vue-storefront/pull/6353/commits/41c9831c9ec2be830f7db5fbd44acd84c959f8b8)